packages/storybook/stories/va-radio-uswds.stories.jsx

```diff
@@ -0,0 +1,325 @@
+import React, { useState, useEffect } from 'react';
+import { VaRadio, VaRadioOption} from '@department-of-veterans-affairs/web-components/react-bindings';
+
+import {
+  getWebComponentDocs,
+  componentStructure,
+  propStructure,
+  StoryDocs,
+} from './wc-helpers';
+
+VaRadio.displayName = 'VaRadio';
+VaRadioOption.displayName = 'VaRadioOption';
+
+const radioDocs = getWebComponentDocs('va-radio');
+const radioItem = getWebComponentDocs('va-radio-option');
+
+export default {
+    title: `USWDS/Radio button USWDS`,
+    id: 'uswds/va-radio',
+  subcomponents: componentStructure(radioItem),
+  parameters: {
+    componentSubtitle: `va-radio web component`,
+    docs: {
+      page: () => <StoryDocs data={radioDocs} />,
+    },
+  },
+};
+
+const vaRadioConst = args => {
+  const {   
+    'enable-analytics': enableAnalytics,
+    error,
+    label,
+    hint,
+    required,
+    tile,
```

Comment:
Do we need title in this example?

Reply:
Yes, I think I can clean that up in a few places from the story. Thanks!